### ISSUE TITLE
Discuss themes for Website 

### ISSUE THREAD

**Issue by eshandhawan51** [wildfirepy/wildfirepy.org#2]
This issue is opened to keep track of all the theme related discussions and all the appropriate sources found.

**Comment by kakirastern** [comment]
So I am thinking @Raahul-Singh would like a theme quite like the SunPy website theme which can be found at https://sunpy.org/. Their repo is at https://github.com/sunpy/sunpy.org. We should definitely have some color theme picked out first.

**Comment by eshandhawan51** [comment]
The site looks good and would suit the project as well 
related to colour theme 
I thought of using something like #f88909 
and a hint of brown 
what say @Raahul-Singh @kakirastern

**Comment by kakirastern** [comment]
Yeah. I like the idea of #f88909 plus a hint of brown? How about adding forest green to the mix (#228b22) to make it a theme of three colors?

**Comment by eshandhawan51** [comment]
it would give a great effect considering that the thing is related to forest fires

**Comment by eshandhawan51** [comment]
If we have to follow sunpy's theme 
We need a background image related to our context

**Comment by kakirastern** [comment]
Yeah, I will task you with finding a suitable background image then
